mmcv/ops/csrc/onnxruntime/cpu/roi_align_rotated.cpp

```diff
@@ -2,6 +2,7 @@
 // https://github.com/facebookresearch/detectron2/tree/master/detectron2/layers/csrc/ROIAlignRotated
 // Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
 #include "roi_align_rotated.h"
+
```

Comment:
This modification may not be necessary in this PR.

Reply:
Simply fix the lint error in master branch